package.json

```diff
@@ -2,55 +2,63 @@
   "name": "recipes-client",
   "version": "1.0.0",
   "description": "",
-  "main": "src/index.js",
   "scripts": {
     "test": "jest",
     "test:dev": "jest --watch",
-    "start": "parcel watch recipes-client/index.tsx --out-dir public/recipes-client",
-    "build": "parcel build recipes-client/index.tsx --out-dir public/recipes-client",
+    "start": "parcel watch recipes-client/index.tsx --dist-dir public/recipes-client",
+    "build": "parcel build recipes-client/index.tsx --dist-dir public/recipes-client",
     "lint": "eslint recipes-client --ext .ts,.tsx",
     "format": "prettier --write recipes-client",
     "fill-db": "node scripts/fill.js"
   },
   "author": "",
   "license": "ISC",
+  "browserslist": [
+    "since 2017-06"
+  ],
   "devDependencies": {
-    "@types/file-saver": "^2.0.2",
-    "@types/html-parser": "^0.11.0",
-    "@types/jest": "^26.0.22",
-    "@types/node": "^14.14.20",
-    "@typescript-eslint/eslint-plugin": "^3.9.1",
-    "@typescript-eslint/parser": "^3.9.1",
-    "aws-sdk": "^2.860.0",
-    "eslint": "^7.7.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-emotion": "^10.0.27",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.0",
-    "jest": "^26.4.2",
-    "parcel-bundler": "^1.12.4",
-    "prettier": "2.0.5",
-    "ts-jest": "^26.3.0",
-    "typescript": "^4.0.2"
+    "@emotion/eslint-plugin": "^11.11.0",
+    "@guardian/eslint-config-typescript": "^7.0.0",
+    "@guardian/eslint-plugin-source-foundations": "^15.0.0",
+    "@guardian/eslint-plugin-source-react-components": "^19.0.0",
+    "@guardian/prettier": "^5.0.0",
+    "@guardian/tsconfig": "^0.2.0",
+    "@types/file-saver": "^2.0.5",
+    "@types/html-parser": "^0.11.3",
+    "@types/jest": "^29.5.5",
+    "@types/node": "^20.6.3",
+    "@typescript-eslint/eslint-plugin": "^6.7.2",
+    "@typescript-eslint/parser": "^6.7.2",
+    "aws-sdk": "^2.1462.0",
+    "eslint": "^8.49.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-emotion": "^11.0.0",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^29.7.0",
```

Comment:
Bumping jest causes `Error: Cannot find module 'ts-jest/utils'` build error.
https://stackoverflow.com/questions/72960399/ts-jest-utils-missing

Probably need to change this line https://github.com/guardian/recipes/blob/1815de7c0e39e4a0e64bcf012d50b95e50355271/jest.config.js#L1